blog/_posts/2017-08-01-one-month-of-RGSoC.md

```diff
@@ -0,0 +1,42 @@
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">It was so nice to meet our incredible supervisor <a href="https://twitter.com/ines_opcoelho">@ines_opcoelho</a> !! We had a meeting today with her and all our team :D <a href="https://t.co/Ep0gpb3DYS">https://t.co/Ep0gpb3DYS</a></p>&mdash; Rails Gyn (@RailsGyn) <a href="https://twitter.com/RailsGyn/status/882023345478414336">July 3, 2017</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+*Our teams are having fun with kick-off calls & their projects [Credit: RGSoC Teams]*
+
+One amazing thing to note this year: we have teams from Sri Lanka (Team Fusion), Albania (Team Codeaholics) and Russia (Team Alexa) for the first time! If you’re interested in knowing where the other teams are from, check out [this](https://railsgirlssummerofcode.org/blog/2017-05-22-teams-2017) post. 
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Yesterday, we had our <a href="https://twitter.com/RailsGirlsSoC">@RailsGirlsSoC</a> kick off celebration! 🎉So our coaches joined to help us planning our project &amp; finish eating the cake 😁 <a href="https://t.co/65jDBAYQ8L">pic.twitter.com/65jDBAYQ8L</a></p>&mdash; Team Fusion (@TeamFusionJK) <a href="https://twitter.com/TeamFusionJK/status/882101855861460992">July 4, 2017</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+*Team Fusion on their kick off call celebration [Credit: Team Fusion]*
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Been at Pydata Moscow meetup today, they wrote &quot;RGSOC&quot; on our badges. Not sure if it qualifies as a kick-off party, but it was nice. <a href="https://t.co/rzybOP2dLD">pic.twitter.com/rzybOP2dLD</a></p>&mdash; Team Alexa (@linguistsare) <a href="https://twitter.com/linguistsare/status/878379842697285632">June 23, 2017</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+*Team Alexa at PyData Moscow meetup [Credit: Team Alexa]*
+
+Looking at the progress of our teams, this seems to be a wonderful start of the program. Stay tuned for blog posts from our teams to follow their amazing journey and checkout their daily logs on our [Teams app](https://teams.railsgirlssummerofcode.org/).
+
+![A month of RGSoC!](/img/blog/2017/2017-rgsoc-team_collage.jpg)<font color="grey"><small><i>Our 2017 teams :) (Image: Laura Gaetano)</i></small></font>
```

Comment:
could you add here the `<div class="image-credits"></div>` instead of `<font color="grey">...`? A lot of the old posts still have that old version and need to be changed eventually, but the style has been “refactored” as an `.image-credits` class which makes it much easier to use for everyone 😇